backend/api/routes/cart.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express')
+const Cart = require('../models/Cart.model')
+const auth = require('../../auth')
+const router = express.Router()
+
+// Make new cart
+router.post('/', auth.verifyCustomer, async (req, res, next) => {
```

Comment:
We don't need this route to create a cart. A cart is automatically created when a user is created